### PR TITLE
Bump dcos-net

### PR DESCRIPTION
**Pull request by urbanserj**
## High-level description

Bump dcos-net

## Corresponding DC/OS tickets (obligatory)

These DC/OS JIRA ticket(s) must be updated (ideally closed) in the moment this PR lands:

  - [DCOS_OSS-4328](https://jira.mesosphere.com/browse/DCOS_OSS-4328) Lashup sometime fails to converge
  - [DCOS_OSS-4970](https://jira.mesosphere.com/browse/DCOS_OSS-4970) Conflict between VIP port and port mapping

## Related tickets (optional)

Other tickets related to this change:

  - [DCOS_OSS-5006](https://jira.mesosphere.com/browse/DCOS_OSS-5006) Add support for ipset protocol to gen_netlink library

## Checklist for all PRs

  - [x] Added a comprehensible changelog entry to `CHANGES.md` or explain why this is not a user-facing change: all fixes are back ported into 1.13 branch and will be released soon
  - [x] Included a test which will fail if code is reverted but test is not. If there is no test please explain here: none
  - [x] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [x] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [x] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [x] Test Results: [link to CI job test results for component]
  - [x] Code Coverage (if available): [link to code coverage report]
https://github.com/dcos/dcos-net/compare/35506bb4d6a8e8b0b19234012ae5b9089d72bf0a...d660500895765c9f966a997ec66779677cc5477b
https://github.com/dcos/dcos-net/pull/138
https://github.com/mesosphere/gen_netlink/pull/20
https://github.com/dcos/dcos-net/pull/139
https://github.com/dcos/lashup/pull/50
https://codecov.io/github/dcos/dcos-net/commit/d660500895765c9f966a997ec66779677cc5477b

___
**PLEASE FILL IN THE TEMPLATE ABOVE** / **DO NOT REMOVE ANY SECTIONS ABOVE THIS LINE**

## Instructions and review process

**What is the review process and when will my changes land?**

All PRs require 2 approvals using GitHub's [pull request reviews](https://help.github.com/articles/about-pull-request-reviews/).

Reviewers should be:
* Developers who understand the code being modified.
* Developers responsible for code that interacts with or depends on the code being modified.

It is best to proactively ask for 2 reviews by @mentioning the candidate reviewers in the PR comments area. The responsibility is on the developer submitting the PR to follow-up with reviewers and make sure a PR is reviewed in a timely manner. Once a PR has **2 ship-it's**, **no red reviews**, and **all tests are green** it will be included in the [next train](https://github.com/dcos/dcos/blob/master/contributing.md).

### CODE REVIEW

File: packages/dcos-integration-test/extra/test_networking.py [hunk 1]
```diff
@@ -271,6 +271,47 @@ def test_vip_ipv6(dcos_api_session):
                     marathon.Network.USER, marathon.Network.USER, ipv6=True)
 
 
+@pytest.mark.slow
+@pytest.mark.parametrize(
+    'container',
+    list(marathon.Container))
+def test_vip_port_mapping(dcos_api_session,
```
Comment (GoelDeepak) [inline packages/dcos-integration-test/extra/test_networking.py]: isn't `port_mapping` get tested as part of `test_vip` BRIDGE mode?

Comment (urbanserj) [reply packages/dcos-integration-test/extra/test_networking.py]: It's true, but it's another case. `test_vip` checks connectivity between two tasks, `origin` and `proxy`. Here we start the third task `pm` with port mapping on the host where `proxy` is running. For example,

1. `origin` task is running on 10.0.0.2 with `1.1.1.7:10001` VIP label
2. `proxy` task is running on 10.0.0.1
3. `pm` task is running on 10.0.0.1 on bridge network with port mapping, host port 10001

It means we have `1.1.1.7:10001` (vip) and `10.0.0.1:10001` (pm) on the same host and `1.1.1.7:10001` must always be our `origin` task.

Comment (GoelDeepak) [reply packages/dcos-integration-test/extra/test_networking.py]: Sorry, I missed the fact that this test was added to support the fix. Thanks for explaining it.

**Comment by urbanserj** [comment]
@mesosphere-mergebot label [Ready For Review]

File: gen/dcos-config.yaml [hunk 2]
```diff
@@ -1826,6 +1842,12 @@ package:
               {
                   "Command": ["iptables-save"]
               },
+              {
+                  "Command": ["ip6tables-save"]
+              },
+              {
+                  "Command": ["ipset list"]
```
Comment (janisz) [inline gen/dcos-config.yaml]: `ipset` is not installed on our CI.
Please add a test for it

File: gen/dcos-config.yaml [hunk 3]
```diff
@@ -1826,6 +1842,12 @@ package:
               {
                   "Command": ["iptables-save"]
               },
+              {
+                  "Command": ["ip6tables-save"]
```
Comment (janisz) [inline gen/dcos-config.yaml]: Please add a test for this

File: gen/dcos-config.yaml (hunk 2 above)
Comment (urbanserj) [reply gen/dcos-config.yaml]: It's installed, but there is a mistake in my code 🤦‍♂️

```suggestion
                  "Command": ["ipset", "list"]
```

File: gen/dcos-config.yaml (hunk 3 above)
Comment (urbanserj) [reply gen/dcos-config.yaml]: https://github.com/dcos/dcos/pull/5170

I'm curious, why didn't tests fail on this PR?

Comment (urbanserj) [reply gen/dcos-config.yaml]: @janisz the test has passed in #5170, but how can we be sure that the issue is fixed if it was not failing consistently before?

**Pull request merged**